go/logic/ccql.go

```diff
@@ -23,7 +23,7 @@ func queryHost(host string, user string, password string, defaultSchema string,
 			return err
 		}
 		for _, row := range resultData {
-			output := []string{host}
+			output := []string{host, schema}
```

Comment:
This is breaking existing behavior: this change forces the addition of the schema column. Perhaps omit the schema column if only a single schema is specified? I'm wondering whether there should be a command line flag to show/hide schema?

Reply:
When working with multi sharded environment, the visibility of the schema column becomes an asset. But I agree this can be a breaking change, and I can try to add a command line flag to view the schema and only if that flag is set, the schema column becomes visible.